frontend/src/store/objectStore.ts

```diff
@@ -80,16 +80,27 @@ export const useObjectStore = defineStore('objectStore', () => {
         if (uniqueIds.length) {
           objects = (await objectService.listObjects({ objId: uniqueIds, ...params })).data;
           const metadataResponse = (await objectService.getMetadata(null, { objId: uniqueIds })).data;
+          const taggingResponse = (await objectService.getObjectTagging({ objId: uniqueIds })).data;
 
           objects.forEach(async (obj: any) => {
             const metadata = metadataResponse.find((x: Metadata) => x.objectId === obj.id);
-            // TODO: Tags
 
             if (metadata) {
               obj.metadata = metadata;
+              obj.metadata.metadata.sort(
+                (metadata1: any, metadata2: any ) =>
+                  metadata1.key < metadata2.key ? -1 : metadata1.key > metadata2.key ? 1 : 0
+              );
               obj.name = metadata.metadata.find((x: { key: string }) => x.key === 'name')?.value;
             }
 
+            if(taggingResponse) {
+              obj.tag = taggingResponse.find((x: ObjectTag) => x.objectId === obj.id);
+              obj.tag.tagset.sort(
+                (tag1: Tag, tag2: Tag) => tag1.key < tag2.key ? -1 : tag1.key > tag2.key ? 1 : 0
+              );
+            }
+
```

Comment:
We should consider during this string sort comparison func into a util function. Not a worry for this PR, but I will probably refactor this sprint if I get to some of those sorting bugs.